config/config_test.go

```diff
@@ -77,7 +77,12 @@ func TestWriteFile(t *testing.T) {
 		"reconnectPeriod": "10s",
 		"nickname": ""
 	},
-	"net": ""
+	"net": "",
+	"metrics": {
+		"enabled": false,
+		"reportInterval": "3s",
+		"prometheusEndpoint": "/ip4/0.0.0.0/tcp/8888"
```

Comment:
What would it look like to support another (or multiple) monitoring system? Opencensus seems to support many exporters. If we were to, say, support Graphite, would we add a `graphiteEndpoint`? If so, we would either need to set the endpoint to nil to indicate it's enabled (and no enable flag) or have separate enable flags for each endpoint.

My preference would be the latter:
``` go
"metrics": {
		"reportInterval": "3s",
		"prometheusEnabled": false,
		"prometheusEndpoint": "/ip4/0.0.0.0/tcp/8888"
		"graphiteEnabled": true,
		"graphiteEndpoint": "/ip4/0.0.0.0/tcp/2003"
}
```

If you agree, maybe we should rename `enabled` to `prometheusEnabled` now.

Reply:
This is definitely worth considering down the road, but I think it may be outside the scope of this PR, we don't use graphite anywhere so I would prefer not to add configuration options for it here. Is that alright with you?

Reply:
Non-blocking: the suggestion is just to rename the enabled flag to prometheusEnabled to future proof the config. It's not a big deal.